### DISCUSSION TITLE
System theme

### DISCUSSION THREAD

**Discussion by xmha97** [cefsharp/CefSharp#3953] [Q&A] [answered]
Lively Wallpaper uses CefSharp
CefSharp doesn't support OS theme

https://user-images.githubusercontent.com/23485114/148655713-3858cbb9-1eb4-4665-a21d-75dd0dcfff50.mp4

**Comment by amaitland** [comment, marked as answer]
Dark mode is supported you can test using https://github.com/cefsharp/CefSharp.MinimalExample

As CefSharp is only the browser control, styling the rest of your application is up to you.

> Lively Wallpaper uses CefSharp
CefSharp doesn't support OS theme

In future please ask a question.
